_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -4468,13 +4468,13 @@
 /turf/simulated/floor/plating,
 /area/maintenance/fsmaint)
 "amf" = (
-/obj/structure/closet/secure_closet/security,
 /obj/structure/window/reinforced{
 	dir = 4
 	},
 /obj/item/radio/intercom{
 	pixel_y = 25
 	},
+/obj/machinery/vending/secdrobe,
```

Comment:
Does sec really need *two* clothing vendors? I know fashion is the main focus of the department, but this is quite a lot of clothes for maybe a dozen people.
![image](https://user-images.githubusercontent.com/57483089/128427516-f8c39d83-0b38-4e58-8209-66283b74abf2.png)
![image](https://user-images.githubusercontent.com/57483089/128428030-bb832d25-e5d6-465f-8429-05a03d90dac5.png)

Reply:
Two were placed as to not destroy the symmetry of the equipment room as before this there was a corporate security clothing wardrobe and a normal security one, didn't want to keep just one wardrobe there when the vendor had it all anyway so I added two.